### DISCUSSION TITLE
Axis Marker on plot

### DISCUSSION THREAD

**Discussion by Tyskanchik** [ra3xdh/qucs_s#604] [Ideas]
The existing marker is not convenient, everyone has to enter a value, etc.
As a solution, I suggest adding an axis marker, which is present in most other programs.
For example:
> [2024-02-21-11-05-28.webm](https://github.com/ra3xdh/qucs_s/assets/70572897/652898c7-c566-489d-a817-360f649964fa)

**Comment by ra3xdh** [comment]
Totally agreed with your proposal. This was asked and discussed several times before. I am also finding the cursors like in the digital oscilloscope more convenient than existing marker. But the adding of the proposed feature may require a deep redesign of the plotting subsystem. I have no time to perform this task myself. So don't expect this to be implemented in the near years.

FIY: markers could be moved with arrow keys on keyboard. Arrow left/right along X-axis direction. Arrow up/down change trace on par.sweep plot.
